File: 2017-05-05/entities/patient.py
from helpers.helpers import convert_blood_type as conv

class Patient:
	def __init__ (self, fio, age, gender, blood_type=1, Rh="+"):
		self.fio = fio
		self.age = age
		self.gender = gender
		self.blood_type = blood_type
		self.Rh = Rh
		print "Patient created."

	def __str__(self):
		return "%s (%s%s)"  % (self.fio, conv(self.blood_type), self.Rh )

File: 2017-11-16/Patient.java
public class Patient {
	private int id;
	private String fio;
	private int year;
	private char gender;
	
	public Patient(int id, String fio, int year, char gender) {
		this.id = id;
		this.fio = fio;
		this.year = year;
		this.gender = gender;
	}	
	
	public String toString() {
		return fio + " : " + year + " : " + gender;
	}

	public int getId() {
		return id;
	}
		
}